internal/controller/datadogagent/merger/rbac.go

```diff
@@ -246,7 +251,7 @@ func (m *rbacManagerImpl) AddClusterRoleBinding(namespace string, roleName strin
 	bindingObj, _ := m.store.GetOrCreate(kubernetes.ClusterRoleBindingKind, "", roleName)
 	clusterRoleBinding, ok := bindingObj.(*rbacv1.ClusterRoleBinding)
 	if !ok {
-		return fmt.Errorf("unable to get from the store the ClusterRoleBinding %s/%s", namespace, roleName)
+		return fmt.Errorf("unable to get from the store the ClusterRoleBinding %s", roleName)
```

Comment:
nit: not directly related to the PR, but improving the log: there's no namespace for a clusterrole